docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -282,3 +282,64 @@ DSA keys look almost identical but begin with ``ssh-dss`` rather than
 
     :raises cryptography.exceptions.UnsupportedAlgorithm: If the serialized
         key is of a type that is not supported.
+
+Serialization Formats
+~~~~~~~~~~~~~~~~~~~~~
+
+.. class:: Format
+
+    .. versionadded:: 0.8
+
+    An enumeration for key formats.
```

Comment:
Probably specifically link to `as_bytes`, though that's going to get bloaty when we have 6 classes to link to.

(Let's include the link for now and revisit as we add more implementations)